fdbrpc/include/fdbrpc/DDSketch.h

```diff
@@ -228,7 +228,10 @@ class DDSketch : public DDSketchBase<DDSketch<T>, T> {
 		return ceil(fastLogger::fastlog(sample) * multiplier) + offset;
 	}
 
-	T getValue(size_t index) { return fastLogger::reverseLog((index - offset) / multiplier) * 2.0 / (1 + gamma); }
+	T getValue(size_t index) {
```

Comment:
Would using int64_t all around here have addressed the issue? https://google.github.io/styleguide/cppguide.html#Integer_Types

(signed vs. unsigned is a surprisingly contentious topic. I'm a firm believer that ints should almost always be signed though)

Reply:
Yeah I think it would have fixed the issue in this case. I think it just seems strange to allow an index to be negative however.

Reply:
It looks like it actually used to be signed, and was changed in 92dccdd9eb

Reply:
I can go ahead and change it back to signed.

Reply:
We should assert that errorGuarantee will not create a buckets vector so large that an `int` cannot index it.